src/Components/SavedSearchAlert/Components/PriceRangeFilter.tsx

```diff
@@ -0,0 +1,31 @@
+import { Expandable, Spacer } from "@artsy/palette"
+import { PriceRange } from "Components/PriceRange/PriceRange"
+import {
+  CustomRange,
+  DEFAULT_PRICE_RANGE,
+} from "Components/PriceRange/constants"
+import { useSavedSearchAlertContext } from "Components/SavedSearchAlert/SavedSearchAlertContext"
+import { FC } from "react"
+
+export const PriceRangeFilter: FC = () => {
+  const { criteria, setCriteriaValue } = useSavedSearchAlertContext()
+
+  const handlePriceRangeUpdate = (updatedRange: CustomRange) => {
+    setCriteriaValue("priceRange", updatedRange.join("-"))
+  }
+
+  return (
+    <Expandable
+      label="Set price range you are interested in"
```

Comment:
**suggestion / non-blocking**: Minor but we could consider putting this copy in the locale file. Example [here (copy)](https://github.com/artsy/force/blob/92683c40ea6e84f9e35bbf5fba82c63e0bede278/src/System/i18n/locales/en-US/translation.json#L87) and [here (use)](https://github.com/artsy/force/blob/92683c40ea6e84f9e35bbf5fba82c63e0bede278/src/Apps/Artwork/Components/ArtworkSidebar/ArtworkSidebarCreateArtworkAlert.tsx#L89).